src/app/modules/analytics/analytics.service.ts

```diff
@@ -73,7 +74,7 @@ export const getFormCount = (): ResultAsync<number, DatabaseError> => {
         error,
       })
 
-      return new DatabaseError()
+      return new DatabaseError('Error retrieving form collection count')
```

Comment:
can use the `getMongoErrorMessage` utility to get a string to pass to DatabaseError